src/sentry/status_checks/celery_alive.py

```diff
@@ -15,9 +15,22 @@ def check(self):
         last_ping = options.get('sentry:last_worker_ping') or 0
         if last_ping >= time() - 300:
             return []
+
+        backlogged, size = None, 0
```

Comment:
It's cleaner if used this way. It's used in the error message on that level of scope.